packages/block-library/src/image/style.scss

```diff
@@ -11,8 +11,8 @@
 		text-align: center;
 	}
 
-	&.alignfull img,
-	&.alignwide img {
+	&.alignfull > img,
```

Comment:
I suppose it's better to add specificity to here to be specific about the inheritance, than accidentally override the width on an image in a caption.

Reply:
The selector specificity remains the same though (0,2,1), so a theme using `.wp-block-image.alignfull img´ will still override this like it used to.

Reply:
How do you calculate specificity? This is not something I've learned yet, eager to glean a trick!

Reply:
Playing with https://specificity.keegan.st/ explains it by testing pretty well. Excluding inline styles and `!important`:

1. IDs
2. Classes, attributes, pseudo classes (:hover)
3. Elements and pseudo elements (:before, :after)

So `#id .wp-block-image.is-large img` is (1,2,1). To override it you need to either match or override something on the left side. Eg these would override:

- `#id .wp-block-image.is-large img` (1,2,1)
- `#id#id img` (2,0,1)

These would not:

- `.wp-block-image.wp-block-image.wp-block-image.wp-block-image.is-large img` (0,5,1)

And `div img` (0,0,2) would be overridden by eg `[src]` (0,1,0)

Reply:
Nice. Thank you!